llvm/tools/llvm-snippy/README.md

```diff
@@ -243,8 +260,44 @@ You can reproduce the same generation by repeating the call with the same seed.
+
+Thank you for showing interest in contributing to LLVM-snippy. These are several ways you can help to make the generator better.
+
+## Bug Reports
 
-For questions regarding llvm-snippy and its development, as well as your suggestions, please contact konstantin.vladimirov@syntacore.com
+We want to know about all LLVM-snippy bugs: segmentation faults, poor or incorrect diagnostics, lack of randomization in the generated test, incorrectly working feature, etc. If you think that you have encountered a bug, please file an issue on GitHub. Make sure that you provide minimal description, reproduction and HEAD commit hash in the issue.
+
+## Bug Fixes
+
+We appreciate your desire to improve LLVM-snippy. You can start working on any open unassigned issue. A comment in the issue is enough to show that you have started working on it. If you do not have bandwidth to work on the issue already assigned to you, please let others know by leaving a comment in the issue, so it can be reassigned.
+If the fix requires design discussion please create an [RFC](#driving-a-major-feature). Anyway, we encourage everyone to provide a short summary in the issue comments before creating a PR.
+
+## Driving a Major Feature
+
+If you want to introduce a major change or implement a new feature in LLVM-snippy, please create an RFC first. We would like to keep LLVM-snippy community informed about major changes. We should also reach consensus on all technical and design decisions before any significant work is done.
+There is no template for an RFC, feel free to create it on your own. Though usually a good RFC contains: overview, proposal, pros and cons, implementation steps.
+
+## Backward compatibility
+
+We try to keep backward compatibility in snippy for our customers. Backward incompatible changes includes, but not limited to removal of an existing option, change of input configuration format, amends in the signatures of entry functions or globals. These changes require major release version change and must be disscussed in prior. Please, create an [RFC](#driving-a-major-feature) for such changes.
+
+## Submitting a Patch
+
+When your patch is ready for review, create pull-request. We ask you to follow rules:
+1. Adhere [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html) where applicable
+2. Run git-clang-format on your changes
+3. Remove any unrelated changes from your patch
+4. Add at least one lit-test if possible
+5. Make a single commit
+6. Create a PR from your fork or use a branch that starts from `users/<username>/`
+
+You can request review by mentioning people in the PR comments. Best candidates for review are developers who have contributed in the area your patch touches. Be aware that a normal period for review is one week, so please do not ping more often.
+
+When you patch is reviewed and got approval you can merge the change. If you do not have rights, please let people know so they can merge the change on your behalf.
```

Comment:
```suggestion
Once you patch is reviewed and approved, you can merge the change. If you do not have the rights, please let people know so they could merge the change on your behalf.
```

Reply:
Also, i would definitely elaborate a bit on "people" - who are they? Other reviewers? Developers? Contributors? Otherwise, it's very confusing.

Reply:
Maintainers.